server/src/main/java/org/opensearch/common/cache/store/enums/CacheStoreType.java

```diff
@@ -8,11 +8,14 @@
 
 package org.opensearch.common.cache.store.enums;
 
+import org.opensearch.common.annotation.ExperimentalApi;
+
 /**
  * Cache store types in tiered cache.
  *
  * @opensearch.internal
  */
+@ExperimentalApi
 public enum CacheStoreType {
```

Comment:
Do we need to enumerate the possible tiers based on how they're stored? Can't tiers just be numbers? 

The implementation of a particular cache decides where it's stored.

Reply:
I don't think I understood that correctly.
This general stores info about the store type available now within caches. Now, a tiered cache for example, can be comprised of onHeap or onDisk OR anyone within OpenSearch(like IndicesRequestCache) can also use onHeap independently.